src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -14,7 +14,7 @@ public function assertResponseOk()
 	{
 		$actual = $this->response->getStatusCode();
 
-		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' .$actual);
+		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' . $actual);
```

Comment:
@shehi Since you mentioned PSR guidelines, you probably know how important a consistent code style is for projects as large as this one.

Your PR looks good except for this small issue - it would be awesome if you could fix this. In fact, you can also interpolate the variables directly in a string with double quotes - that's also fine by Laravel's standards:

```php
"Expected status code 200, got $actual"
```